### DISCUSSION TITLE
so-kratos [ ERROR]

### DISCUSSION THREAD

**Discussion by yashar6909** [Security-Onion-Solutions/securityonion#1728] [Unsupported Versions]
I've updated with $sudo soup and also did a $sudo yum update
Now, if type $sudo so-status so-kratos is not available, and as an analyst machine, I cannot reach NGINX (I get a 500 error).
Is there a way to fix this?

Thanks
yashar

**Comment by TOoSmOotH** [comment]
try so-docker-refresh and then state.highstate

**Reply by yashar6909** [reply]
sudo so-docker-refresh
Please run soup on the manager. The manager controls all updates.

Then ran sudo soup, however everything was up to date
